romancal/datamodels/schemas/reference_files/flat.schema.yaml

```diff
@@ -0,0 +1,25 @@
+%YAML 1.1
+---
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/flat.schema"
+allOf:
+- $ref: referencefile.schema
+- $ref: ../core.schema
```

Comment:
Another option is to `$ref` in the core schema from within referencefile.schema.  Advantage is not having to remember to include core in the individual schemas for each reference type.  Disadvantage is a deeper hierarchy of schemas, which can be harder to think about.  It's a matter of taste, but I thought I'd better point out that the option exists.

Reply:
Done.